bucket/grepwin.json

```diff
@@ -0,0 +1,65 @@
+{
+    "version": "1.9.1",
+    "description": "Regular expression search and replace tool for Windows.",
+    "homepage": "https://tools.stefankueng.com/grepWin.html",
+    "license": "GPL-3.0-only",
+    "notes": "Run 'reg import $dir\\grepwin-install-context.reg' to add grepWin to right-click context menu. \n(This does not require admin rights)",
```

Comment:
Why removing `reg import`?

Opening the `.reg` file directly with regedit will require admin rights.
`reg import xxx.reg` does not require admin rights. (when only registries under `HKCU` are modified)
This is useful when the user does not have an admin account (e.g. using a PC at the workplace)

Also, opening `.reg` file directly will show up a message box. (even with admin rights)
If you are running all the `scoop` installs with a simple batch file, this will interrupt the whole process.
I think it will be better to not showing the message box at all.

Reply:
If you are running in batch there is no difference to execute it from explorer or opening new terminal to run this command.

Reply:
okay, I will remove the "reg import", for the sake of simplicity.